### ISSUE TITLE
Add active buttons states

### ISSUE THREAD

**Issue by tnordberg** [scimusmn/tsck-cochlear-implant#17]
Add semi-transparent "active" states to all the buttons as seen here: https://d.pr/i/EVWXtt

**Comment by abdulrauf11** [comment]
@tnordberg Just to clarify, when you click on a button it should become semi-transparent? Should it revert to the original state once you lift your finger?

**Comment by tnordberg** [comment]
@abdulrauf11 that's correct. Only when it's being pressed. 
This behavior should align with the css `:active` selector.

**Comment by abdulrauf11** [comment]
@tnordberg Are the white circles behind the icons also a part of button? I mean do they become faded as well once clicked?

**Comment by tnordberg** [comment]
@abdulrauf11 yes, in this interface the white circles should be considered part of the button and also fade.

Good question– in this instance, I think we can retain both button states, even though they are visually similar. I think they both will read independently without causing any confusion.
